client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -85,6 +95,41 @@ module.exports = React.createClass( {
 		return info;
 	},
 
+	handleDomainVolumeSelection: function( event ) {
+		event.preventDefault();
+		let volume = parseInt( event.target.value );
+		upgradesActions.setVolume( this.props.cartItem, volume );
+
+		this.recordEvent( 'changeVolume', volume );
+	},
+
+	getVolumeOptions: function() {
+		return range( 1, 6 ).map( number =>
```

Comment:
This is just to make sure you want five numbers `1, 2, 3, 4, 5` printed here. It looks like you know what you're doing, but it's a bit sneaky here the way `range()` goes up-to-but-not-over the end value.

A comment would greatly help here, or since the list is so small it could be inlined (though I still prefer to use `range()` for its clarity)

```js
return [1, 2, 3, 4, 5].map( number => … );

return range( 1, 5 + 1 ).map( number => … );

// present 1-year to 5-year subscriptions
return range( 1, 6 ).map( number => … );
```